src/module_type_translator.ts

```diff
@@ -209,83 +208,56 @@ export class ModuleTypeTranslator {
   }
 
   /**
-   * Returns the `const x = goog.forwardDeclare...` text for an import of the given `importPath`.
+   * Records that we we want a `const x = goog.requireType...` import of the given `importPath`,
+   * which will be inserted when we emit.
    * This also registers aliases for symbols from the module that map to this forward declare.
+   *
+   * @param isExplicitImport True if this comes from an underlying 'import' statement, false
+   *     if this reference is needed just because a symbol's type relies on it.
+   * @param isDefaultImport True if the import statement is a default import, e.g.
+   *     `import Foo from ...;`, which matters for adjusting whether we emit a `.default`.
    */
-  forwardDeclare(
+  requireType(
       importPath: string, moduleSymbol: ts.Symbol, isExplicitImport: boolean,
       isDefaultImport = false) {
     if (this.host.untyped) return;
     // Already imported? Do not emit a duplicate forward declare.
-    if (this.forwardDeclaredModules.has(moduleSymbol)) return;
+    if (this.requireTypeModules.has(moduleSymbol)) return;
     if (this.host.typeBlackListPaths && this.host.typeBlackListPaths.has(importPath)) {
-      return;  // Do not emit goog.forwardDeclare or goog.require for blacklisted paths.
+      return;  // Do not emit goog.requireType for blacklisted paths.
     }
     const nsImport = googmodule.extractGoogNamespaceImport(importPath);
-    const forwardDeclarePrefix = `tsickle_forward_declare_${++this.forwardDeclareCounter}`;
+    const requireTypePrefix = `tsickle_forward_declare_${this.additionalImports.length}`;
```

Comment:
Given that we don't care much about the var name, consider `length + 1` to minimize the test diff.